src/schema/__tests__/city.test.ts

```diff
@@ -150,6 +150,20 @@ describe("City", () => {
       expect(gravityOptions).toMatchObject({ displayable: true })
       expect(gravityOptions).not.toHaveProperty("discoverable")
     })
+    it("requests shows with location, by default", async () => {
+      await runQuery(query, context)
+      const gravityOptions = context.showsWithHeadersLoader.mock.calls[0][0]
+
+      expect(gravityOptions).toMatchObject({ has_location: true })
+      expect(gravityOptions).not.toHaveProperty("discoverable")
+    })
+    it("requests shows not at a fair, by default", async () => {
```

Comment:
Suggested wording tweak to convey the intent a little more clearly:

```suggestion
    it("excludes fair booths, by default", async () => {
```

Reply:
nice! haha i wonder if i can commit this suggestions in one commit!

Reply:
Oh yeah, I guess each one of these suggestions kicks off a new build. Feel free to ignore the GH suggestions and push up your own commit too!

Reply:
🙌 updated